examples/isentropic_example.py

```diff
@@ -0,0 +1,215 @@
+#
+# Once three dimensional data in isobaric coordinates has been pulled and the desired
+# isentropic levels created, the conversion to isentropic coordinates can begin. Data will be
+# passed to the function as below. The function requires that isentropic levels, isobaric
+# levels, and temperature be input. Any additional inputs (in this case relative humidity, u,
+# and v wind components) will be linearly interpolated to isentropic space.
+
+isent_anal = mcalc.isentropic_interpolation(isentlevs,
+                                            lev,
+                                            tmp,
+                                            spech,
+                                            uwnd,
+                                            vwnd,
+                                            hgt,
+                                            tmpk_out=True)
+
+#####################################
+# The output is a list, so now we will separate the variables to different names before
+# plotting.
+
+
+isentprs = isent_anal[0]
+isenttmp = isent_anal[1]
+isentspech = isent_anal[2]
+isentu = isent_anal[3].to('kt')
+isentv = isent_anal[4].to('kt')
+isenthgt = isent_anal[5]
+
+########################################
+# A quick look at the shape of these variables will show that the data is now in isentropic
+# coordinates, with the number of vertical levels as specified above.
+
+print(isentprs.shape)
+print(isentspech.shape)
+print(isentu.shape)
+print(isentv.shape)
+print(isenttmp.shape)
+print(isenthgt.shape)
+
+#################################
+# **Converting to Relative Humidity**
+#
+# The NARR only gives specific humidity on isobaric vertical levels, so relative humidity will
+# have to be calculated after the interpolation to isentropic space.
+
+isentrh = mcalc.relative_humidity_from_specific_humidity(isentspech, isenttmp, isentprs)
+
+#######################################
+# **Plotting the Isentropic Analysis**
+
+
+# Set up our projection
+crs = ccrs.LambertConformal(central_longitude=-100.0, central_latitude=45.0)
+
+
+# Set up our array of latitude and longitude values and transform to
+# the desired projection.
+tlatlons = crs.transform_points(ccrs.PlateCarree(), lon, lat)
+tlons = tlatlons[:, :, 0]
+tlats = tlatlons[:, :, 1]
+
+# Limit the extent of the map area, must convert to proper coords.
+LL = crs.transform_point(-122., 22., ccrs.PlateCarree())
+UR = crs.transform_point(-60., 47., ccrs.PlateCarree())
+
+# Choose a level to plot, in this case 296 K
+level = 0
+
+# Get data to plot state and province boundaries
+states_provinces = cfeature.NaturalEarthFeature(category='cultural',
+                                                name='admin_1_states_provinces_lakes',
+                                                scale='50m',
+                                                facecolor='none')
+
+fig = plt.figure(1, figsize=(17., 12.))
+ax = plt.subplot(111, projection=crs)
+ax.set_extent([LL[0], UR[0], LL[1], UR[1]], crs)
```

Comment:
That works for me. Do you know why the bounds of the plot change when setting directly in ax.set_extent? It looks to me like it should give exactly the same plot.